src/java/io/bazel/rulesscala/scalac/reporter/DepsTrackingReporter.java

```diff
@@ -17,6 +17,9 @@
 import java.util.Set;
 import java.util.jar.JarFile;
 import java.util.stream.Collectors;
+
+import javax.print.attribute.standard.Severity;
```

Comment:
Looks to be unused import

Reply:
@liucijus:  ok. That unused import is removed.
Fixed.